private/src/scripts/editor/blocks/post-list/components/editable/LinkList.jsx

```diff
@@ -14,7 +15,10 @@ const LinkItem = (props) => {
 
   return (
     <li>
-      <article className="linkList-item">
+      <article
+        className="linkList-item"
+        style={{ backgroundImage: `url(${props.featured_image})` }}
```

Comment:
if there's no featured image set, this will still output `style="background-image: url()"` in the DOM.

Reply:
Added conditional to check featured_image exists before outputting the background image styles